Source/WebKit/GPUProcess/graphics/RemoteDisplayListRecorder.messages.in

```diff
@@ -55,6 +55,7 @@ messages -> RemoteDisplayListRecorder NotRefCounted Stream {
     DrawSystemImage(Ref<WebCore::SystemImage> systemImage, WebCore::FloatRect destinationRect)
     DrawPattern(WebCore::RenderingResourceIdentifier imageIdentifier, WebCore::FloatRect destRect, WebCore::FloatRect tileRect, WebCore::AffineTransform transform, WebCore::FloatPoint phase, WebCore::FloatSize spacing, struct WebCore::ImagePaintingOptions options)
     BeginTransparencyLayer(float opacity)
+    BeginTransparencyLayerWithCompositeMode(struct WebCore::CompositeMode compositeMode)
```

Comment:
This is called beginTransparencyLayer in RemoteDisplayListRecorder. So we should either rename it here, or if same name is not supported in IPC messages, rename the method name in the class.